modules/markup/orgmode/orgmode.go

```diff
@@ -51,7 +51,7 @@ func (Renderer) SanitizerRules() []setting.MarkupSanitizerRule {
 // Render renders orgmode rawbytes to HTML
 func Render(ctx *markup.RenderContext, input io.Reader, output io.Writer) error {
 	htmlWriter := org.NewHTMLWriter()
-	htmlWriter.HighlightCodeBlock = func(source, lang string, inline bool) string {
+	htmlWriter.HighlightCodeBlock = func(source, lang string, inline bool, params map[string]string) string {
```

Comment:
The function is changed in https://github.com/niklasfasching/go-org/commit/9b56fc914c80f6b8a6f6dde9c19d70ff7e79a89b